app/src/main/java/crux/bphc/cms/fragments/MyCoursesFragment.java

```diff
@@ -443,13 +446,18 @@ public void setDownloadClickListener(ClickListener downloadClickListener) {
                 });
 
                 more_options.setOnClickListener(view -> {
-                    MoreOptionsFragment.OptionsViewModel moreOptionsViewModel = MyCoursesFragment.this.moreOptionsViewModel;
-                    Observer<MoreOptionsFragment.Option> observer;  // to handle the selection
-                    //Set up our options and their handlers
-                    ArrayList<MoreOptionsFragment.Option> options = new ArrayList<>();
+                            MoreOptionsFragment.OptionsViewModel moreOptionsViewModel = MyCoursesFragment.this.moreOptionsViewModel;
+                            Observer<MoreOptionsFragment.Option> observer;  // to handle the selection
+                            //Set up our options and their handlers
+                            ArrayList<MoreOptionsFragment.Option> options = new ArrayList<>();
+
+                            String favouriteOption = ( mCourseList.get(getLayoutPosition()).isFavorite() ? "Remove from favourites" : "Add to favourites");
+                            boolean isFavorite = (!mCourseList.get(getLayoutPosition()).isFavorite());
```

Comment:
Indentation is wrong

Reply:
Why is there a not operator on the rvalue?

Reply:
Consider moving this line above line `String favouriteOption = ( mCourseList.get(getLayoutPosition()).isFavorite() ...`. That way, you can replace the ternary operator's condition with the `isFavorite` variable.